data/projects/anon-klub.ts

```diff
@@ -7,7 +7,7 @@ AnonKlub is a tool designed for Ethereum developers that allows for anonymous pr
 export const anonKlub: ProjectInterface = {
   id: "anon-klub",
   section: "pse",
-  projectStatus: "active",
+  projectStatus: "inactive",
```

Comment:
btw (out of scope of this PR), have you considered using an enum for this?
```suggestion
  projectStatus: ProjectStatus.Inactive,
```

which would be defined in [`@/lib/types`](https://github.com/privacy-scaling-explorations/pse.dev/blob/05076fb21301af433bea843adfaf938b9da711af/lib/types.ts#L12-L13)

As a dev not familiar with this repo, I had to look for a project which I knew was inactive and check for its data to know what exact string to use. No big deal, but an enum would force doing the right thing automatically.

Reply:
@kalidiagne would you mind add a little description for this?

Reply:
can just add more detail in this readme:https://github.com/privacy-scaling-explorations/pse.dev/blob/main/data/projects/README.md

Reply:
@ChialiT , will add more details for this thank you!